AEPIdentity/Sources/IdentityState.swift

```diff
@@ -145,14 +145,17 @@ class IdentityState {
     ///   - response: the response data if any
     ///   - eventDispatcher: a function which when invoked dispatches an `Event` to the `EventHub`
     ///   - createSharedState: a function which when invoked creates a shared state for the Identity extension
-    func handleHitResponse(hit: IdentityHit, response: Data?, eventDispatcher: (Event) -> Void, createSharedState: ([String: Any], Event?) -> Void) {
+    ///   - createXDMSharedState: a function which when invoked creates a XDM shared state for the Identity extension
+    func handleHitResponse(hit: IdentityHit, response: Data?, eventDispatcher: (Event) -> Void, createSharedState: ([String: Any], Event?) -> Void,
+                           createXDMSharedState: ([String: Any], Event?) -> Void) {
```

Comment:
This would not be required if this path is never hit after the migration.

Reply:
Good catch, makes sense given my other PR now.